.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to this toolbox
+
+We welcome all contributions that help us achieve our aim of speeding up ML/AI research in health and life sciences.
+
+Examples of contributions are
+
+* Data loaders for specific health & life sciences data
+* Network architectures and components for deep learning models
+* Tools to analyze and/or visualize data
+
+All contributions to the toolbox need to come with unit tests, and will be reviewed when a Pull Request (PR) is started.
+If in doubt, reach out to the core `hi-ml` team before starting your work.
+
+Please look through the existing folder structure to find a good home for your contribution.
+
+For a full set of design and coding guidelines, please refer to either
+
+* The published guidelines on [readthedocs](https://hi-ml.readthedocs.io/en/latest/contribution_guidelines.html)
+* The latest guidelines [in the repository](https://github.com/microsoft/hi-ml/blob/main/docs/source/coding_guidelines.md)
```

Comment:
Aren't these the same? I think there should be a single source of information for this.